tests/pivots/spreadsheet_pivot/spreadsheet_pivot.test.ts

```diff
@@ -660,6 +660,33 @@ describe("Spreadsheet Pivot", () => {
     );
   });
 
+  test("sorted PIVOT include_total=FALSE with no column groups includes column data", () => {
```

Comment:
note: this test was passing before the fix thanks to the combination of both "off-by-one" errors